python/ray/scripts/scripts.py

```diff
@@ -747,8 +750,18 @@ def start(
         cli_logger.newline()
         with cli_logger.group("Next steps"):
             dashboard_url = node.address_info["webui_url"]
-            if bootstrap_address.startswith("127.0.0.1:"):
-                if ray_constants.ENABLE_RAY_CLUSTER:
+            if ray_constants.ENABLE_RAY_CLUSTER:
+                cli_logger.print("To add another node to this Ray cluster, run")
```

Comment:
This is now only printed when you want to start a multi-node cluster. This means by default you won't see this on OSX and windows unless you set the `RAY_ENABLE_WINDOWS_OR_OSX_CLUSTER` env.

Reply:
what's happening if you run ray start --address in win/mac? Does it raise an exception to set this env var?

Reply:
I didn't change that behavior. Yes it will abort and raise an exception.

Reply:
I was also thinking whether we should check the value of address, if it's 127.0.0.1 we proceed (basically saying multi-node Ray cluster on a single machine is supported) otherwise we raise an exception to set this env var.